clients/common/utility.cpp

```diff
@@ -189,8 +192,8 @@ rocblas_operation char2rocblas_operation(char value)
     case 'n': return rocblas_operation_none;
     case 't': return rocblas_operation_transpose;
     case 'c': return rocblas_operation_conjugate_transpose;
+    default: return static_cast<rocblas_operation>(-1);
```

Comment:
Rather than silently falling back on defaults, a value of -1 (cast to the appropriate enum) is returned for an unknown value. This is used in other places (e.g. client.cpp) to detect illegal option values, rather than having to test it in two places.